src/api/test/functional/webui/spider_test.rb

```diff
@@ -53,6 +53,7 @@ def getlinks(baseuri, body)
   def raiseit(message, url)
     # known issues
     return if url =~ %r{/package/binary/BinaryprotectedProject/.*}
+    return if url =~ %r{/package/binary/download/*}
```

Comment:
This probably changed because we now always show the link, while before it was only visible to logged in users. We should either hide the link like it was before or disable it when the user is not logged in.

That way we also avoid that bots click non-working links.

Reply:
Hm, that sucks.

Thanks for explaining